src/rouge.jl

```diff
@@ -0,0 +1,152 @@
+#= ROUGE score implementation
+Lin C.Y. , 2004
+Rouge: A package for automatic evaluation of summaries
+Proceedings of the workshop on text summarization branches out (WAS 2004) (2004), pp. 25-26
+Link to paper:
+http://www.aclweb.org/anthology/W04-1013 =#
+
+using TextAnalysis
+using WordTokenizers
```

Comment:
This is already there in TextAnalysis.jl, please remove this line and the one above this.